### PR TITLE
Add an osx build

### PR DESCRIPTION
**Pull request by posix4e**

### CODE REVIEW

**Comment by posix4e** [comment]
This also uses pg_embed for the osx build. Happy to do the linux in another commit, but I wanna make sure I'm using it correctly @solidsnack

File: .travis.yml [hunk 1]
```diff
@@ -16,4 +15,4 @@ addons:
     - libclang-common-3.4-dev
     - clang-3.4
 
-script: util/travis
+script: util/travis_${TRAVIS_OS_NAME}.sh
```
Comment (solidsnack) [inline .travis.yml]: Since these scripts are not Boune shell, it would be better to drop `.sh` (no point in adding extra info that is not accurate).

**Comment by solidsnack** [comment]
I will try this out later today.

**Comment by posix4e** [comment]
No rush. I'm getting some incredible build interference on travi's osx builds.

**Comment by posix4e** [comment]
updated should pass now @solidsnack

**Comment by posix4e** [comment]
This code doesn't seem to work on 9.5

**Pull request merged**